src/main/java/net/openhft/chronicle/core/pool/ClassAliasPool.java

```
@@ -190,6 +192,22 @@ private Class<?> doLookup(String name) {
         }
     }
 
+    private boolean banned(String name) {
+        if (name.isEmpty())
```

why would name be empty?

---

It shouldn't, that's why it's banned

---

should we throw an exception or assert instead?

---

I could leave this code out, but you would get an IndexOutOfBoundException which could be confusing.

---

All banned class names result in a specific ClassNotFoundRuntimeException